RELEASES.md

```diff
@@ -68,6 +68,11 @@ Cargo
   included in backups or content indexing (on Windows).
 - Added automatic `@` argfile support, which will use "response files" if the
   command-line to `rustc` exceeds the operating system's limit.
+  
+Rustdoc
+-------
+
+- [Doctests are now run on unexported `macro_rules!`, matching other private items][96630]
```

Comment:
We may want to put this in "Compatibility Notes" instead of as a new feature?

```suggestion
- [Doctests are now run on unexported `macro_rules!` macros, matching other private items][96630]
```

Reply:
Yeah, I think it works better as a compatibility note - r=me with that done.

Reply:
Moved it.